docs/cli/new.md

```diff
@@ -39,25 +44,53 @@ pulls a nio project from the `bar` repository of a GitHub user `foo` and names i
 
 ---
 
+## Example (configure Pubkeeper):
+
+```bash
+nio new my_project --pubkeeper-hostname <system_id>.pubkeeper.nio.works --pubkeeper-token <pubkeeper_token>
+```
+
+creates a new folder in your current directory named `my_project` with pubkeeper settings configured for system wide communication.
+
+---
+
+## Example (configure authentication):
+
+```bash
+nio new my_project --username <username> --password <password>
+```
+
+creates a new folder in your current directory named `my_project` with a specified user created to access this instance. Use your configured `<username>` and `<password>` values in the [System Designer](/system-designer/reference.md#connect-to-a-local-instance).
+
+---
+
+## Example (configure encryption):
+
+```bash
+nio new my_project --ssl
+```
+
+creates a new folder in your current directory named `my_project`. Follow the prompts to configure your instance encryption
```

Comment:
Is this example showing me that my new project will have ssl enabled, or disabled? I realize that that explanation is in new.md, but it would be good to clairfy here, as well.